src/marshmallow/__init__.py

```diff
@@ -18,14 +19,49 @@
 
 from . import fields
 
-# TODO: Deprecate __version__?
-__version__ = importlib.metadata.version("marshmallow")
-__parsed_version__ = Version(__version__)
-__version_info__: tuple[int, int, int] | tuple[
-    int, int, int, str, int
-] = __parsed_version__.release  # type: ignore[assignment]
-if __parsed_version__.pre:
-    __version_info__ += __parsed_version__.pre  # type: ignore[assignment]
+
+def __getattr__(name: str) -> typing.Any:
+    import warnings
```

Comment:
I suppose this is imported here to avoid a useless import in normal use.
I don't mind either way.

Reply:
Oh, OK. Strange at first sight, but alright.